.devcontainer/Dockerfile

```
@@ -16,12 +16,12 @@ RUN apt-get update && apt-get install dos2unix git python2 curl -y \
     && cd /features \
     # locking to the latest master commit in this repo (https://github.com/devcontainers/features.git) to prevent breaking changes
     # We should update this commit hash from time to time to
```

```suggestion
    # We should update this commit hash from time to time to
```
missing end of sentence 😅